skill/tests/mock_alice.py

```diff
@@ -0,0 +1,316 @@
+        self.slots.clear()
+
+    @chained
+    def repeat(self):
+        self.name = "YANDEX.REPEAT"
+        self.slots.clear()
+
+
+class AliceEntity:
+    def __init__(self, type=""):
+        self.type = type
+        self.tokens = {"start": 0, "end": 0}
+        self.value = ""
+
+    @chained
+    def tokens(self, start=0, end=0):
+        self.tokens["start"] = start
+        self.tokens["end"] = end
+
+    @chained
+    def value(self, value):
+        self.value = value
+
+    @chained
+    def fio(self, first_name="", patronymic_name="", last_name=""):
+        assert first_name or patronymic_name or last_name
+
+        self.type = "YANDEX.FIO"
+        self.value = {
+            "first_name": first_name,
+            "patronymic_name": patronymic_name,
+            "last_name": last_name,
+        }
+
+    @chained
+    def number(self, num: int):
+        self.type = "YANDEX.NUMBER"
+        self.value = num
+
+    @chained
+    def datetime(
+        self,
+        year="",
+        year_is_relative=False,
+        month="",
+        month_is_relative=False,
+        day="",
+        day_is_relative=False,
+        hour="",
+        hour_is_relative=False,
+        minute="",
+        minute_is_relative=False,
+    ):
+        self.type = "YANDEX.DATETIME"
+        self.value = {
+            "year": year,
+            "year_is_relative": year_is_relative,
+            "month": month,
+            "month_is_relative": month_is_relative,
+            "day": day,
+            "day_is_relative": day_is_relative,
+            "hour": hour,
+            "hour_is_relative": hour_is_relative,
+            "minute": minute,
+            "minute_is_relative": minute_is_relative,
+        }
+
+    def val(self):
+        return {"type": self.type, "value": self.value}
+
+
+class AliceRequest:
+    def __init__(self):
+        self.new_session = False
+        self.command = ""
+        self.original_utterance = ""
+        self.nlu_tokens = []
+        self.intents = {}
+        self.entities = []
+        self.state_sessions = {}
+        self.state_user = {}
+        self.state_application = {}
+
+    @chained
+    def new_session(self):
+        self.new_session = True
+
+    @chained
+    def set_command(self, command=""):
+        self.command = command
+        self.set_original_utterance(command)
+
+    @chained
+    def set_original_utterance(self, original_utterance=""):
+        self.original_utterance = original_utterance
+        self.set_nlu_token(original_utterance.lower().split(" "))
+
+    @chained
+    def set_nlu_token(self, tokens: list):
+        self.nlu_tokens = tokens
+
+    @chained
+    def add_intent(self, intent: AliceIntent):
+        self.intents.update(intent.val())
+
+    @chained
+    def add_entity(self, entity: AliceEntity):
+        self.entities.append(entity.val())
+
+    @chained
+    def add_to_state_session(self, name: str, value):
+        self.state_sessions.update({name: value})
```

Comment:
```suggestion
        self.state_sessions[name] = value
```
Для добавления одного значения в словарь update выглядит громоздко.